infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -377,7 +377,7 @@ private CorfuRuntime getRuntime(ServerContext serverContext) {
             .tlsEnabled((Boolean) serverContext.getServerConfig().get("--enable-tls"))
             .systemDownHandler(() -> System.exit(SYSTEM_EXIT_ERROR_CODE))
             .maxCacheEntries(serverContext.getLogReplicationCacheMaxSize()/2)
-            .maxWriteSize(serverContext.getMaxWriteSize())
+            .maxUncompressedWriteSize(serverContext.getMaxWriteSize())
```

Comment:
I think we should have a check to make sure we don't go over the default maxUncompressedWriteSize.